File: app/src/main/java/com/tebet/mojual/view/qualityreject/OrderRejectFragmentProvider.java
package com.tebet.mojual.view.qualityreject;

import com.tebet.mojual.view.qualitydetail.OrderDetailFragmentModule;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class OrderRejectFragmentProvider {

    @ContributesAndroidInjector(modules = OrderDetailFragmentModule.class)
    abstract OrderRejectFragment provideQualityFragmentFactory();
}

File: app/src/main/java/com/tebet/mojual/view/qualityreject/OrderRejectFragmentProvider.kt
package com.tebet.mojual.view.qualityreject

import com.tebet.mojual.view.qualitydetail.OrderDetailFragmentModule

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class OrderRejectFragmentProvider {

    @ContributesAndroidInjector(modules = [OrderDetailFragmentModule::class])
    internal abstract fun provideQualityFragmentFactory(): OrderRejectFragment
}
